michaelmsinclair-toronto.md

```diff
@@ -0,0 +1,8 @@
+# Toronto, Ontario, Canada
+
+### Restaurants
```

Comment:
I have been to Toranto, but I didn't find any good Ice-Cream places.  Is Ice-Cream big up there?